src/utilities/i18n/tests/I18n.test.ts

```diff
@@ -35,7 +35,16 @@ describe('I18n.translate', () => {
     expect(() => {
       i18n.translate('key', {notString: 'bar'});
     }).toThrow(
-      `No replacement found for key 'string'. The following replacements were passed: 'notString'`,
+      `Error in translation for key 'key'. No replacement found for key 'string'. The following replacements were passed: '{"notString":"bar"}'`,
+    );
+
```

Comment:
Not 100% necessary but it might be best to split these 2 expect statements into 2 tests?